Vagrantfile

```diff
@@ -78,6 +78,7 @@ Vagrant.configure(2) do |config|
       grant all privileges on *.* to 'manoseimas'@'localhost';
       flush privileges;" || true
     sudo apt-get install -y nodejs nodejs-legacy npm
+    sudo apt-get install -y coffeescript
```

Comment:
CoffeeScript should be already installed using `npm install manoseimas/widget/frontend/` from https://github.com/ManoSeimas/manoseimas.lt/blob/master/manoseimas/widget/frontend/package.json with pinned versions.